R/api.R

```diff
@@ -489,6 +490,23 @@ endpoint_report_version_artefact_hashes <- function(path) {
     returning = returning_json("Artefacts.schema"))
 }
 
+target_reports <- function(path, reports = NULL) {
+  if (!is.null(reports)) {
+    reports <- unlist(strsplit(reports, split = ","))
+  }
+  db <- orderly::orderly_db("destination", root = path)
+  get_all_reports(db, reports)
+}
+
+endpoint_reports <- function(path) {
+  porcelain::porcelain_endpoint$new(
+    "GET", "/v1/reports/",
+    target_reports,
+    porcelain::porcelain_input_query(reports = "string"),
```

Comment:
Does comma separated string here make more sense than just making this a POST? do we do the same elsewhere? I have a feeling that either porcelain does, or we intended that it could, posting multiple copies of a query parameter to concatenate into a vector...